modules/040-control-plane-manager/openapi/config-values.yaml

```diff
@@ -6,6 +6,32 @@ properties:
     description: |
       `kube-apiserver` parameters.
     properties:
+      admissionPlugins:
+        type: array
+        description: |
+          Enable additional [admission plugins](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers).
+
+          Always enabled admission plugins in addition to the ones enabled by Kubernetes itself:
+          1. ExtendedResourceToleration
+          2. EventRateLimit with the following config:
+
+          apiVersion: eventratelimit.admission.k8s.io/v1alpha1
+          kind: Configuration
+          limits:
+          - type: Namespace
+            qps: 50
+            burst: 100
+            cacheSize: 2000
+        x-examples:
+          - [ "AlwaysPullImages", "NamespaceAutoProvision" ]
+        items:
+          type: string
+          enum:
+          - AlwaysPullImages
+          - NamespaceAutoProvision
+          - OwnerReferencesPermissionEnforcement
+          - PodNodeSelector
```

Comment:
As said in the doc, the PodNodeSelector admission controller also demands config.
https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podnodeselector

Reply:
Nope, the doc is ambiguous. And I've verified that this is not the case by reading the source [code](https://github.com/kubernetes/kubernetes/blob/f0ea54070bec90dd829b7054117d670f9f90839f/plugin/pkg/admission/podnodeselector/admission.go#L74-L97) (it get an element from a nil map, but that's fine, it just returns an empty string). It'll work just with annotations.

Reply:
Thus you could not provide defaults, but annotations work just fine? That's good news!

Could you please add a comment to the open API spec with the link and the description you have written?

Reply:
Done.